### PR TITLE
Installs Coveralls Metrics

### PR DESCRIPTION
**Pull request by zanenkn**
[PT Story](https://www.pivotaltracker.com/story/show/166112603)

## Changes proposed in this PR:

* Installs and configures Coveralls gem

### CODE REVIEW

File: coverage/.last_run.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (oliverochman) [inline coverage/.last_run.json]: add coverage to the `.gitignore`

**Review by oliverochman — CHANGES_REQUESTED** [review]
I **need** to see some git ponging, the same person has committed and created the PR the last pull requests.

File: coverage/.last_run.json (hunk 1 above)
Comment (raptorf1) [reply coverage/.last_run.json]: added

Comment (raptorf1) [reply coverage/.last_run.json]: boa will continue

**Review by oliverochman — APPROVED** [review]
Good job guys!

**Pull request merged**